app/views/about/index.html.erb

```diff
@@ -29,11 +29,11 @@
               </thead>
               <tbody>
               <% @smart_proxies.each do |proxy| %>
-                <tr>
+                <tr class="proxy-show"  data-url="<%= ping_smart_proxy_path(proxy) %>">
                   <td><%= link_to_if_authorized proxy.name, hash_for_edit_smart_proxy_path(proxy) %></td>
                   <td><%=h proxy.features.to_sentence %></td>
-                  <td><div class="proxy-status" data-url=<%= ping_smart_proxy_path(proxy) %>><%= _("Connecting..") %></div></td>
-                  <td><div class="proxy-version" data-url=<%= version_smart_proxy_path(proxy) %>><%= _("Connecting..") %></div></td>
+                  <td><div class="proxy-show-status"><%= spinner %></div></td>
+                  <td><div class="proxy-version"><%= spinner %><</div></td>
```

Comment:
It looks like there's some mistake here? When the spinner is still loading, a open tag character shows up next to it. 
![screenshot from 2015-12-28 12-44-50](https://cloud.githubusercontent.com/assets/598891/12018268/ef103b3e-ad60-11e5-841a-6bc4696a111a.png)